SwiftyAccordionCells/ViewController.swift

```diff
@@ -66,7 +73,7 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
         } else if (item.isHidden) {
             return 0
         } else {
-            return 44
+            return UITableViewAutomaticDimension
```

Comment:
This change allows for the self-sizing cells and dynamic type support using auto layout.